kubernetes/machine_classes/gcp-machine-class.yaml

```diff
@@ -26,6 +26,7 @@ spec:
   networkInterfaces:
   - network: network-name # Network name to attach the instance to
     subnetwork: sub-net-name # Subnet name to attach the instance to
+    disableExternalIP: true # If external IP is not required for the NIC, set this to true
```

Comment:
I'd suggest, we either keep this `false` in the documentation or rather mention in the comment about the consequences of node-joining.
- There is a chance of bad UX otherwise if someone tries MCM independently.

Reply:
Done.